### PR TITLE
Add note about processing of anti-forgery tokens in header and form

### PR DESCRIPTION
**Pull request by captainsafia**
Clarifying the behavior of antiforgery validation when a token is provided in both the header and form payload.

---

#### Internal previews

| 📄 File | 🔗 Preview link |
|:--|:--|
| [aspnetcore/security/anti-request-forgery.md](https://github.com/dotnet/AspNetCore.Docs/blob/730cbc5a1a9f8da0e3c54b7072414510b8e6b32d/aspnetcore/security/anti-request-forgery.md) | [Prevent Cross-Site Request Forgery (XSRF/CSRF) attacks in ASP.NET Core](https://review.learn.microsoft.com/en-us/aspnet/core/security/anti-request-forgery?branch=pr-en-us-30090) |

### CODE REVIEW

File: aspnetcore/security/anti-request-forgery.md [hunk 1]
```diff
@@ -263,6 +263,9 @@ The following example uses JavaScript to make an AJAX request to obtain the toke
 
 <a name="antimin7"></a>
 
+> [!NOTE]
+> When the antiforgery token is provided in both the request header and in the form payload, only the token in the header will be validated.
```
Comment (guardrex) [inline aspnetcore/security/anti-request-forgery.md:267]:
```suggestion
> When the antiforgery token is provided in both the request header and in the form payload, only the token in the header is validated.
```

File: aspnetcore/security/anti-request-forgery.md [hunk 2]
```diff
@@ -536,6 +539,9 @@ The following example uses JavaScript to make an AJAX request to obtain the toke
...
```
Comment (guardrex) [inline aspnetcore/security/anti-request-forgery.md:543]:
```suggestion
> When the antiforgery token is provided in both the request header and in the form payload, only the token in the header is validated.
```

File: aspnetcore/security/anti-request-forgery.md [hunk 3]
```diff
@@ -1213,6 +1219,9 @@ public void ConfigureServices(IServiceCollection services)
 }
 ```
 
+> [!NOTE]
+> When the antiforgery token is provided in both the request header and in the form payload, only the token in the header will be validated.
```
Comment (guardrex) [inline aspnetcore/security/anti-request-forgery.md:1223]:
```suggestion
> When the antiforgery token is provided in both the request header and in the form payload, only the token in the header is validated.
```

**Review by guardrex — APPROVED** [review]
Just a NIT on present tense.

File: aspnetcore/security/anti-request-forgery.md (hunk 1 above)
Comment (Rick-Anderson) [inline aspnetcore/security/anti-request-forgery.md:267]:
```suggestion
> When the antiforgery token is provided in both the request header and in the form payload, only the token in the header is validated.
```

File: aspnetcore/security/anti-request-forgery.md (hunk 2 above)
Comment (Rick-Anderson) [inline aspnetcore/security/anti-request-forgery.md:543]:
```suggestion
> When the antiforgery token is provided in both the request header and in the form payload, only the token in the header is validated.
```

File: aspnetcore/security/anti-request-forgery.md (hunk 3 above)
Comment (Rick-Anderson) [inline aspnetcore/security/anti-request-forgery.md:1223]:
```suggestion
> When the antiforgery token is provided in both the request header and in the form payload, only the token in the header is validated.
```

**Review by Rick-Anderson — APPROVED** [review]
After removing FT (future tense)

**Review by Rick-Anderson — APPROVED** [review]

**Comment by captainsafia** [comment]
TY for following up on this @Rick-Anderson!

**Pull request merged**